src/smokeTest/java/uk/gov/hmcts/reform/draftstore/SmokeTestSuite.java

```diff
@@ -76,7 +79,9 @@ public void tearDown() {
      * @return Idam access token
      */
     private String signIntoIdam(IdamClient idamClient) {
-        Optional<String> authorisationCode = idamClient.getAuthorisationCode(false);
+        boolean failIfUnauthorised = !useIdamTestingSupport;
+
+        Optional<String> authorisationCode = idamClient.getAuthorisationCode(failIfUnauthorised);
```

Comment:
I think it would be better to have tests behave the same way on all envs if possible...

Can we stop using 'testing-support' altogether?

Reply:
Without testing support you need to register a user manually. I've done that in production, but data in other environments is much more short-lived.

Reply:
fair enough